pygmt/tests/test_datasets_earth_magnetic_anomaly.py

```diff
@@ -57,29 +57,13 @@ def test_earth_mag_01d_with_region():
     npt.assert_allclose(data.max(), 127.39996)
 
 
-def test_earth_mag_05m_with_region():
-    """
-    Test loading a subregion of high-resolution earth magnetic anomaly data.
-    """
-    data = load_earth_magnetic_anomaly(
-        resolution="05m", region=[-115, -112, 4, 6], registration="gridline"
-    )
-    assert data.shape == (25, 37)
-    assert data.lat.min() == 4
-    assert data.lat.max() == 6
-    assert data.lon.min() == -115
-    assert data.lon.max() == -112
-    npt.assert_allclose(data.min(), -189.20001)
-    npt.assert_allclose(data.max(), 107)
-
-
-def test_earth_mag_05m_without_region():
+def test_earth_mag_01m_without_region():
     """
     Test loading high-resolution earth magnetic anomaly without passing
     'region'.
     """
     with pytest.raises(GMTInvalidInput):
-        load_earth_magnetic_anomaly("05m")
+        load_earth_magnetic_anomaly("01m")
```

Comment:
`earth_mag` doesn't have `01m` resolution.
```suggestion
        load_earth_magnetic_anomaly("02m")
```